.github/workflows/ci.yml

```diff
@@ -80,13 +84,49 @@ jobs:
             - name: Build other binaries
               run: cargo build --workspace --bins --all-features
 
+    # todo!(linux): Actually run the tests
+    linux_tests:
+        name: (Linux) Run Clippy and tests
+        runs-on: ubuntu-latest
+        steps:
+          - name: Checkout repo
+            uses: actions/checkout@v4
+            with:
+              clean: false
+              submodules: "recursive"
+
+          - name: Restore from cache
+            uses: actions/cache@v3
+            with:
+                path: |
+                  ~/.cargo/bin/
+                  ~/.cargo/registry/index/
+                  ~/.cargo/registry/cache/
+                  ~/.cargo/git/db/
+                  target/
+                key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+                restore-keys: ${{ runner.os }}-cargo-
+
+          - name: Install Rust toolchain
+            uses: dtolnay/rust-toolchain@stable
```

Comment:
> @stable

Just to note, `stable` a floating pointer and might differ from what we use in our rust-toolchain.toml for the rest of the CI: https://github.com/zed-industries/zed/pull/7592/files

Reply:
Good point, I think github action runners also come with the rustup command. I'll see about just removing this...